app.py

```diff
@@ -159,7 +159,18 @@ def news_api_call():
                 'img': news["urlToImage"]
             }
         )
+    socketio.emit('newsData', {
+        'newsObjectLst': newsObjectLst
+    })
+
+    return response_json["articles"]
 
+@socketio.on('news api call')
+def news_api_call():
+    """ sending news back to client """
+    print("Got an event for newz:")
```

Comment:
Im being picky here but probably want to say "Got an event for news" instead of "newz"